pipeline/__init__.py

```diff
@@ -7,6 +7,11 @@
 log = logging.getLogger(__name__)
 
 
+# safe-guard in case `custom` is not provided
+if 'custom' not in dj.config:
+    dj.config['custom'] = {}
```

Comment:
would prefer to chain get so we don't directly modify custom, but will defer for the moment